docs/contributing/testing/naming-conventions.mdx

```diff
@@ -0,0 +1,254 @@
+# ​Naming conventions
+
+:::caution
+
+The information in this document is a work in progress. Decisions need to be discussed and feedback
+needs to be collected before any of the information in here can be considered official.
+
+:::
+
+The main goal of the name is to describe the test in a way that makes it easy to understand what the
```

Comment:
❓ After I read this I got a pretty clear intent for `jest`-based tests but what about our others, like with xUnit? Should this be segmented on framework a bit, and how much can we apply generally but then share framework-specific examples?

Reply:
I had big plans on expanding this document to other languages, working together with @fedemkr to encompass phone apps, and @eliykat and @differsthecat to expand into other areas such as "How to test frontend components" etc. But I always got the feeling that this was a bit outside my responsibilities and other work just got prioritized in the end. @differsthecat talk about testing reminded me of this earlier this week and now here we are :D